6.x/index.md

```diff
@@ -37,6 +37,7 @@
         + [Fields](/docs/{{version}}/crud-fields)
         + [Save Actions](/docs/{{version}}/crud-save-actions)
         + [CrudField JS Library](/docs/{{version}}/crud-fields-javascript-api)
+        + [Custom Validation Rules](/docs/{{version}}/custom-validation-rules)
```

Comment:
I don't think there's a need for a separate page for these custom validation rules. After all... there's not a lot we can say about them (other than... for X use Y, which we'll must already say inside the fields themselves, because that's where people are most likely to stumble onto them).

I don't think these validations are something people will search for _specifically_, so rather than having a separate page for this, I'd add this eithar:

(A) As a section to the **Fields** page. Maybe here?
![CleanShot 2023-04-27 at 10 57 57@2x](https://user-images.githubusercontent.com/1032474/234798287-949f1ad1-e1c8-4c53-9fd9-92d5825a9a25.png)

(B) Mention each CustomValidator inside the docs of the field where it makes sense.

---

Now that I think about it... we should go with B, right?! It's the most intuitive for the developer... and the most unintrusive for the docs. Right?